bin/compile

```diff
@@ -174,9 +174,9 @@ if is_sbt_native_packager $BUILD_DIR || is_play $BUILD_DIR; then
       status "Dropping ivy cache from the slug"
       rm -rf $SBT_USER_HOME/.ivy2
     fi
-    if [ "$KEEP_COURSIER_CACHE" != "true" ] && [ -d $SBT_USER_HOME/.coursier ]; then
+    if [ "$KEEP_COURSIER_CACHE" != "true" ] && [ -d $SBT_USER_HOME/.cache/coursier ]; then
```

Comment:
This will not work as expected since we do delete the whole `.cache` directory from the slug if it exists: https://github.com/heroku/heroku-buildpack-scala/blob/67e2ffdf6a002dd789096ff8ac15eabcc2746d95/bin/compile#L185-L188